apps/synapse-portal-framework/tsconfig.json

```diff
@@ -15,18 +15,14 @@
     "moduleResolution": "bundler",
     "types": ["vite/client"],
     "paths": {
-      "@sage-bionetworks/synapse-types": [
-        "../../../packages/synapse-types/src"
-      ],
+      "@sage-bionetworks/synapse-types": ["../../packages/synapse-types/src"],
       "@sage-bionetworks/synapse-types/*": [
-        "../../../packages/synapse-types/src/*"
-      ],
-      "synapse-react-client": ["../../../packages/synapse-react-client/src"],
-      "synapse-react-client/*": [
-        "../../../packages/synapse-react-client/src/*"
+        "../../packages/synapse-types/src/*"
       ],
-      "vite-config": ["../../../packages/vite-config/src"],
-      "vite-config/*": ["../../../packages/vite-config/src/*"]
+      "synapse-react-client": ["../../packages/synapse-react-client/src"],
+      "synapse-react-client/*": ["../../packages/synapse-react-client/src/*"],
```

Comment:
This should fix the bug; synapse-portal-framework had the wrong path for the synapse-react-client sources. This caused this package to use the build artifacts while the individual portals (using the correct path) used the source code. This led to 2 copies of global objects, such as the React.Context that we store the auth token in. Only one of the authentication contexts was initialized, so consumers of the other context would never get the auth token.

I have no idea why this didn't result in a build-time error :shrug:

Reply:
Nice sleuthing, Nick!  What a difference and superfluous "../" makes!